evm_loader/api/src/api_server/handlers/trace_hash.rs

```diff
@@ -1,30 +1,32 @@
-use crate::NeonApiState;
-use axum::{http::StatusCode, Json};
 use std::convert::Into;
 
-use crate::{context, types::request_models::TraceHashRequestModel};
+use actix_web::{http::StatusCode, post, web, Responder};
+
+use crate::{context, types::request_models::TraceHashRequestModel, NeonApiState};
 
 use super::{parse_emulation_params, process_error, process_result};
 
-#[allow(clippy::unused_async)]
+#[post("/trace-hash")]
```

Comment:
Here must be `trace_hash`, not `trace-hash`. But I'd prefer to rename all routes to the same "hyphen-delimited-format" and temporary add old routes like "trace_hash" for backward compatibility. I think, it could be achieved using methods, which wrap calls to the new methods.